src/app/contextBroker/contextBroker.cpp

```diff
@@ -148,6 +148,7 @@ char            authMech[64];
 char            authDb[64];
```

Comment:
CHANGES_NEXT_RELEASE should include the changes in this PR. Maybe something like this:

```
- Add: CLI parameter - dbUri CLI / env var ORION_MONGO_URI (#3794)
```

Reply:
In addition, documetnation in cli.md should be modified to describe the new CLI and env var

Reply:
Fixed inhttps://github.com/telefonicaid/fiware-orion/pull/4385/commits/d9fd2701511de634009e0d294a8d3093b76ab435 and https://github.com/telefonicaid/fiware-orion/pull/4385/commits/b7cdb855bc95f8d065a1cf8b2ef14286adedb41d